code/app/lagrange.py

```diff
@@ -3,66 +3,61 @@
 import arrayfire as af
 
 from utils import utils
-from scipy import special as sp
 
 from app import global_variables as gvar
 
 def LGL_points(N):
 	"""
-	Returns the N Legendre-Gauss-Laguere points which are
+	Returns the :math: `N` Legendre-Gauss-Laguere points which are
 	the roots of the equation
 	:math::
-		#(1 - x^{2})L'_{N} = 0
+		(1 - x^{2})L'_{N} = 0
 	
 	Parameters
 	----------
 	N : int
-		#Number of LGL-points to be generated.
-		#2 < N < 16
+		Number of LGL-points to be generated.
+		2 < N < 16
 	
 	Returns
 	-------
 	lgl : arrayfire.Array
-		  An array of N LGL points.
+		  An array of :math: `N` LGL points.
 	"""
-	
 	if N > 16 or N < 2:
 		print('Skipping! This function can only return from ',
 			  '2 to 16 LGL points.')
-		pass
+	
 	
 	n = N - 2
 
 	lgl = af.Array(gvar.LGL_list[n])
 	return lgl
 
-def lagrange_basis_coeffs(X):
-    
+
+def lagrange_basis_coeffs(X):    
 	'''
 	A function to get the coefficients of the Lagrange
 	basis polynomials for a given set of values.
 
 	This function calculates the Lagrange basis
 	polynomials by this formula:
 	:math::
-		#L_i = \prod_{m = 0, m \notin i}^{N - 1}
-				#\frac{(x - x_m)}{(x_i - x_m)}
+		L_i = \prod_{m = 0, m \notin i}^{N - 1}
+				\frac{(x - x_m)}{(x_i - x_m)}
 
 	Parameters
 	----------
 	X : numpy array
-		#The :math: `x` coordinates.
+		The :math: `x` coordinates.
 
 	Returns
 	-------
 	numpy.ndarray
```

Comment:
I am just asking you to mention the name of the variable that you are returning in this part of the doc-string.
You need to give a description of what the variable stores.

If you are directly returning an expression then write the mathematical form of the expression that you are returning.